plugins/todo-backend/src/lib/TodoReader/TodoScmReader.ts

```diff
@@ -108,9 +116,11 @@ export class TodoScmReader implements TodoReader {
 
   private async doReadTodos(
     options: ReadTodosOptions,
+    filePathFilter: (filePath: string) => boolean,
```

Comment:
Could skip passing this in here and access `this.filePathFilter` directly tbh, cleans up the code a bit